packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -27,6 +38,34 @@ export function localizePadding(top: number, right: number, bottom: number, left
     };
 }
 
+// TODO: #459: Fix "argument_count"
```

Comment:
lets do this one now - just make it accept a config object instead of a set of arguments.

Reply:
ew thats actually really ugly - maybe leave as is - I think code climate is wrong on this one.